src/Microsoft.ML.TimeSeries/RootCauseAnalyzer.cs

```diff
@@ -10,35 +10,40 @@
 
 namespace Microsoft.ML.TimeSeries
 {
-    public class RootCauseAnalyzer
+    internal class RootCauseAnalyzer
```

Comment:
These classes in this file are currently public, but should really be internal. Can we change that?